draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -280,7 +277,7 @@ At a high level, the Verifier
 
  * receives the `COMBINED-PRESENTATION` from the Holder and verifies the signature of the SD-JWT using the Issuer's public key,
  * verifies the Holder Binding JWT, if Holder Binding is required by the Verifier's policy, using the public key included in the SD-JWT,
- * calculates the hash digests over the Holder-Selected Disclosures and verifies that each digest is contained in the SD-JWT.
+ * calculates the digest s over the Holder-Selected Disclosures and verifies that each digest is contained in the SD-JWT.
```

Comment:
lots of `digest s` that need to be fixed to `digests`

Reply:
fixed